README.md

```diff
@@ -1961,11 +1961,11 @@ bytes.getBytes(/* count */);
 // convert a forge buffer into a Node.js Buffer
 // make sure you specify the encoding as 'binary'
 var forgeBuffer = forge.util.createBuffer();
-var nodeBuffer = new Buffer(forgeBuffer.getBytes(), 'binary');
+var nodeBuffer = Buffer.from(forgeBuffer.getBytes(), 'binary');
 
 // convert a Node.js Buffer into a forge buffer
 // make sure you specify the encoding as 'binary'
-var nodeBuffer = new Buffer();
+var nodeBuffer = Buffer.alloc(10);
```

Comment:
`10` is arbitrary and not easy to understand as an example, let's either adjust this to something that makes more sense, or just remove it.
```suggestion
```

Reply:
what would make more sense?

Reply:
Maybe a sample buffer with some content?

```suggestion
- var nodeBuffer = Buffer.alloc(10);
+ var nodeBuffer = Buffer.from('CAFE', 'hex');
```